lib/CipherInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Aol\Transformers;
+
+interface CipherInterface
+{
+	/**
+	 * Encrypt the given text.
+	 *
+	 * @param string $plain_text The text to encrypt.
+	 *
+	 * @return string The encrypted text.
+	 */
+	function encrypt($plain_text);
```

Comment:
As I was writing the above I was thinking about objects, but it probably does make sense to add a `defineEncryptedArray(...)` as well :)